src/bicep-registry/scheduled-action/test/main.test.bicep

```diff
@@ -0,0 +1,34 @@
+// @resourceGroup @subscription @tenant
+
+targetScope = 'subscription'
+
+// Test 1 - Creating a scheduled alert for the DailyCosts built-in view.
+module dailyCostsAlert '../main.bicep' = {
+  name: 'dailyCostsAlert'
+  params: {
+    name: 'DailyCostsAlert'
+    displayName: 'My schedule'
+    // scope: '/providers/Microsoft.Billing/billingAccounts/8611537' // @tenant
```

Comment:
nit: uncomment

Reply:
This is commented out because it's not supported on subscription scope. This is part of the scope-based templating with the scope directives.